src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -110,6 +111,11 @@ class FrameworkExtension extends Extension
      */
     private $kernelRootHash;
 
+    public function __construct()
+    {
+        require_once __DIR__.'/../Resources/di/tags.php';
```

Comment:
should be outside of the constructor, eg below the ns declaration, or below "use"

Reply:
Out of curiosity, why?

Reply:
because a require has nothing to do in a constructor? how does it relate to the instance?

Reply:
Right, but the idea was to make these constants available only when the extension is instantiated, hence when building the container, not when autoloading the class. But fair enough.